server/backend/api/app/constants.py

```diff
@@ -22,5 +22,11 @@ class AppEnv(str, Enum):
 
 COGNITO_USERNAME_KEY = "username"
 
+FOM_SUBMITTER_ROLE_NAME = "FOM_Submitter"
+FOM_APP_DESC = "Forest Operations Map"
+FOM_ROLE_PURPOSE = "Grant a user access to submit to FOM"
+FOM_APP_DEV_NAME = "fom_dev"
+FOM_APP_TEST_NAME = "fom_test"
```

Comment:
Why do we have app dev and app test names but no prod?

Reply:
I moved tests only constants out from backend constants file to avoid confusion.
Thanks for the catch!